opennn/config.h

```diff
@@ -9,7 +9,9 @@
 
 #define EIGEN_PERMANENTLY_DISABLE_STUPID_WARNINGS
 
-#define _CRT_SECURE_NO_WARNINGS 
+#ifndef _CRT_SECURE_NO_WARNINGS
```

Comment:
Prevents from warnings about duplicate _CRT_SECURE_NO_WARNINGS definitions for almost every file being compiled